core/client/fs/src/main/java/alluxio/client/file/cache/store/LocalPageStoreOptions.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ *
+ */
+
+package alluxio.client.file.cache.store;
+
+import alluxio.Constants;
+
+/**
+ * Options used to instantiate the {@link LocalPageStore}.
+ */
+public class LocalPageStoreOptions extends PageStoreOptions {
+  /**
+   * The root directory where data is stored.
+   */
+  private String mRootDir;
+
+  /**
+   * The max number of buffers used to transfer data. Total memory usage will be equivalent to
+   * {@link #mBufferPoolSize} * {@link #mBufferSize}
+   */
+  private int mBufferPoolSize;
+
+  /**
+   * The size of the buffers used to transfer data. It is recommended to set this at or near the
+   * expected max page size.
+   */
+  private int mBufferSize;
+
+  /**
+   * Creates a new instance of {@link LocalPageStoreOptions}.
+   */
+  public LocalPageStoreOptions() {
```

Comment:
nit: previously in this code base, we used to expose a factory method `defaults()` to create a new instance of options and make the constructor private. just a suggestion as code convention.